config/initializers/feature_toggle.rb

```diff
@@ -20,4 +20,10 @@ def self.location_toggle_enabled?
 
     false
   end
+
+  def self.new_locations_table_layout_enabled?
+    return true unless Rails.env.production?
+
+    false
```

Comment:
i know you're just copying old code (I did the same) but it occurs to me every time i see these they could just be 
```suggestion
    def self.[method_name]?
        !Rails.env.production?
    end
```